docs/pages/account/setup-email.mdx

```diff
@@ -0,0 +1,33 @@
+import { Callout } from "nextra/components";
+import { Steps } from "nextra/components";
+
+# Mein Konto mit einer E-Mail verknüpfen
+
+<Callout type="warning">
+  Die Anmeldung ist aus datenschutzrechtlichen Gründen erst ab 16 Jahren erlaubt!
+</Callout>
+
+Damit deine Daten immer und überall erreichbar sind, musst du deinen Account mit einer E-Mail verknüpfen.  
```

Comment:
```suggestion
Damit deine Daten immer und überall erreichbar sind, musst du deinen Konto mit einer E-Mail verknüpfen.  
```

Let's try to be consistent and mainly use "Konto" instead of "Account"